Sources/Danger/GitLabDSL.swift

```diff
@@ -53,7 +54,12 @@ extension GitLab {
             public let dueDate: Date?
             public let id: Int // swiftlint:disable:this identifier_name
             public let iid: Int
-            public let projectId: Int
+            /// Will has value if it is a regular \
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitLabDSL.swift;
  Line: 57;
-->

- :warning: Line should be 120 characters or less: currently 137 characters (`line_length`)